build_tools/install_rocm_from_artifacts.py

```diff
@@ -189,7 +156,7 @@ def retrieve_artifacts_by_release(args):
     amdgpu_family = args.amdgpu_family
     # In the case that the user passes in latest, we will get the latest nightly-tarball
     if args.release == "latest":
-        release_tag = "nightly-tarball"
+        release_bucket = "therock-nightly-tarball"
```

Comment:
Is "latest" working? I see a TODO above.

If it isn't working, please make the script report a clear error instead of continuing and doing something unexpected or failing later.

To implement latest, especially for dev releases that use git hashes in their versions instead of dates, I think we may need more metadata in our index.